### PR TITLE
Added comment for generateRandomManufacturer method

### PR DESCRIPTION
**Pull request by fabio-sp**
Added comments on random.js method randomManufacturerName() - issue #15

### CODE REVIEW

File: server/simulation/random.js [hunk 1]
```diff
@@ -22,11 +26,12 @@ const randomDroneModel = () => {
   return `${randomManufacturerName()} ${randomModelName()}`;
 };
 
-/**
- * Returns a random rating between 1.0 and 5.0
- *
- * @returns {Number} A floating point number representing a rating between 1.0 and 5.0
- */
+-/**
```
Comment (TalAter) [inline server/simulation/random.js]: Your pull request modifies this unrelated piece of code.
Please make sure your pull request does not modify anything except that which you are trying to fix.

It's a good idea to always check the view of the changes you made before making the pull request to make sure there are no intended side effects.

**Comment by TalAter** [comment]
@fabio-sp please fix the pull request so that when clicking `Files changed` on the top of this page, the only change shown is the one described in the task.

**Comment by fabio-sp** [comment]
Thank you @TalAter. Sorry for all this mess I made.

**Comment by TalAter** [comment]
@fabio-sp Merged!

The final result looks great.

I hope this has served as a good first introduction to contributing to open source for you. If there is one lesson I have learned over a few years of participating in open-source, it is that it pays to be very diligent, exact, and careful... I think you've had a chance to experience this on your first pull request :) I'm sure you'll go on to make some great contributions.

Thank you!

**Pull request merged**